extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1841,10 +1847,12 @@ export class AmpA4A extends AMP.BaseElement {
    * @return {string} full url to safeframe implementation.
    */
   getSafeframePath() {
-    return (
-      'https://tpc.googlesyndication.com/safeframe/' +
-      `${this.safeframeVersion}/html/container.html`
-    );
+    let host = 'tpc.googlesyndication.com';
```

Comment:
You could put this within only doubleclick impl as I am fairly positive we never actually use this in any other extensions of this class (you should verify).

Reply:
moved to `amp-ad-network-doubleclick-impl.js`

Reply:
Actually, in tests  `MockA4AImpl` inherits from `AmpA4A` and parts of `AmpA4A` run that call 'getSafeFramePath`. 

Could keep `getSafeFramePath` in `amp-ad-network-doubleclick-impl` if we also added a fake `getSafeFramePath` to `MockA4AImpl` so the tests still run. 

Or just leave the original `getSafeFramePath` in `AmpA4A`